wolfcrypt/src/integer.c

```diff
@@ -4560,7 +4560,12 @@ static int mp_div_d (mp_int * a, mp_digit b, mp_int * c, mp_digit * d)
      w = (w << ((mp_word)DIGIT_BIT)) | ((mp_word)a->dp[ix]);
 
      if (w >= b) {
+#ifdef WOLFSSL_LINUXKM
+        t = (mp_digit)w;
+        do_div(t, b);
```

Comment:
Where is implementation of do_div?

Reply:
FWIW, for x86-64 it's in /usr/src/linux/arch/x86/include/asm/div64.h.  It's inline asm as a cpp macro.

Reply:
Comment. Thanks!

Reply:
(done, as above.)